src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/xlf/MicrosoftNetCoreAnalyzersResources.pt-BR.xlf

```diff
@@ -2657,26 +2677,6 @@
         <target state="new">Use char literal for a single character lookup</target>
         <note />
       </trans-unit>
-      <trans-unit id="UseStringEqualsOverStringCompareCodeFixTitle">
-        <source>Use 'string.Equals'</source>
-        <target state="new">Use 'string.Equals'</target>
-        <note />
-      </trans-unit>
-      <trans-unit id="UseStringEqualsOverStringCompareDescription">
-        <source>It is both clearer and likely faster to use 'string.Equals' instead of comparing the result of 'string.Compare' to zero.</source>
-        <target state="new">It is both clearer and likely faster to use 'string.Equals' instead of comparing the result of 'string.Compare' to zero.</target>
-        <note />
-      </trans-unit>
-      <trans-unit id="UseStringEqualsOverStringCompareMessage">
-        <source>Use 'string.Equals' instead of comparing the result of 'string.Compare' to 0</source>
-        <target state="new">Use 'string.Equals' instead of comparing the result of 'string.Compare' to 0</target>
-        <note />
-      </trans-unit>
-      <trans-unit id="UseStringEqualsOverStringCompareTitle">
-        <source>Use 'string.Equals'</source>
-        <target state="new">Use 'string.Equals'</target>
-        <note />
-      </trans-unit>
```

Comment:
😕 Why are these moving? They are currently in alphabetical order, but this PR moves them to _not_ be in alphabetical order and also doesn't change the content.

Reply:
lets file a bug on the team